tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/replace_standard_id_in_domain_tables_test.py

```diff
@@ -0,0 +1,258 @@
+                condition_status_concept_id int64
+                >>
+              [(1, 1, 319835, null, null, null, null, null, null, null, null, null, 45567179, null, null),
+               (2, 1, 0, null, null, null, null, null, null, null, null, null, 45567179, null, null),
+               (3, 1, 45567179, null, null, null, null, null, null, null, null, null, 0, null, null),
+               (4, 1, 45567179, null, null, null, null, null, null, null, null, null, 45567179, null, null), 
+               (5, 1, 40398862, null, null, null, null, null, null, null, null, null, 40398862, null, null),
+               (6, 1, 45587397, null, null, null, null, null, null, null, null, null, 45587397, null, null)] col
+        )
+        SELECT 
+            condition_occurrence_id, 
+            person_id, 
+            condition_concept_id, 
+            condition_start_date,
+            condition_start_datetime,
+            condition_end_date,
+            condition_end_datetime,
+            condition_type_concept_id,
+            stop_reason,
+            provider_id,
+            visit_occurrence_id,
+            condition_source_value,
+            condition_source_concept_id,
+            condition_status_source_value,
+            condition_status_concept_id 
+        FROM w, UNNEST(w.col))
+        """)
+
+        mapping_condition_occurrence_data_template = JINJA_ENV.from_string("""
+        DROP TABLE IF EXISTS `{{project_id}}.{{dataset_id}}._mapping_condition_occurrence`;
+        CREATE TABLE `{{project_id}}.{{dataset_id}}._mapping_condition_occurrence`
+        AS (
+        WITH w AS (
+          SELECT ARRAY<STRUCT<condition_occurrence_id int64, src_table_id string, src_dataset_id string, src_condition_occurrence_id int64, src_hpo_id string>>
+              [(1, 'hpo_condition_occurrence', 'hpo_dataset', 10, 'test_hpo'),
+               (2, 'hpo_condition_occurrence', 'hpo_dataset', 20, 'test_hpo'),
+               (3, 'hpo_condition_occurrence', 'hpo_dataset', 30, 'test_hpo'),
+               (4, 'hpo_condition_occurrence', 'hpo_dataset', 40, 'test_hpo'), 
+               (5, 'hpo_condition_occurrence', 'hpo_dataset', 50, 'test_hpo'),
+               (6, 'hpo_condition_occurrence', 'hpo_dataset', 60, 'test_hpo')] col
+        )
+        SELECT condition_occurrence_id, src_table_id, src_dataset_id, src_condition_occurrence_id, src_hpo_id FROM w, UNNEST(w.col))
+        """)
+
+        concept_data_template = JINJA_ENV.from_string("""
+        DROP TABLE IF EXISTS `{{project_id}}.{{dataset_id}}.concept`;
+        CREATE TABLE `{{project_id}}.{{dataset_id}}.concept`
+        AS (
+        WITH w AS (
+          SELECT ARRAY<STRUCT<concept_id int64, concept_name string, standard_concept string>>
+              [(319835, 'Congested Heart Failure', 'S'),
+               (45567179, 'Congested Heart Failure', null),
+               (45587397, 'Anaemia complicating pregnancy, childbirth and the puerperium', null),
```

Comment:
Replaced `null` with `C` here to ensure it is treated the same and the tests ran as expected.

Reply:
Okay will do replace it and then merge.